### PR TITLE
Added GCP tutorial to hubble tutorials

### PR DESCRIPTION
**Pull request by intls**
My Warpcast username: pashu777
Fixes issue #257 

## Why is this change needed?

The addition of the GCP tutorial to the main tutorials section ensures that users can easily find and follow the instructions for setting up a Hubble instance on Google Cloud Platform. This improves the overall user experience and reduces the need to navigate through multiple repositories to find the necessary documentation.
## Summary

This Pull Request adds a tutorial for setting up a Hubble instance on Google Cloud Platform (GCP) to the Hubble tutorials section. The tutorial was previously available in a different location within the project repository, but for consistency and ease of access, it has been added to the main documentation under the tutorials section.

## Changes
Added gcp.md tutorial file to docs/hubble/tutorials/.
Updated tutorials.md to include a link to the new GCP tutorial.
GCP tutorial add

## Merge Checklist

*Choose all relevant options below by adding an `x` now or at any time before submitting for review:*

- [x] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [changeset](https://github.com/atlassian/changesets)
- [x] PR has been tagged with a change label(s) (i.e. documentation, feature, bugfix, or chore)
- [x] PR includes [documentation](https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors) if necessary.

---

## PR-Codex overview
The focus of this PR is to add a tutorial for setting up Hubble on Google Cloud Platform (GCP) with detailed steps and cost information.

### Detailed summary
- Added a new tutorial `gcp.md` for setting up Hubble on GCP
- Includes step-by-step guide, requirements, costs, and configuration instructions
- Utilizes Terraform for creating a GCP VM
- Provides images for visual guidance
- Instructs on installing Docker and verifying Hubble setup

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by intls** [comment]
Closes #257

**Comment by intls** [comment]
@sds Please could you review my commit and pull request!

File: docs/hubble/tutorials.md [hunk 1]
```diff
@@ -3,4 +3,6 @@
 Guides to set up a Hubble instance using various cloud providers.
 
 - [AWS EC2](https://warpcast.notion.site/Set-up-Hubble-on-EC2-Public-23b4e81d8f604ca9bf8b68f4bb086042)
-- [Digital Ocean](https://warpcast.notion.site/Set-up-Hubble-on-DigitalOcean-Public-e38173c487874c91828665e73eac94c1)
+- [Digital Ocean](https://warpcast.notion.site/Set-up-Hubble-on-DigitalOcean-Public-e38173c487874c91828)
```
Comment (sds) [inline docs/hubble/tutorials.md:6]: Why was this Digital Ocean link changed? It's now broken—please restore.

Comment (intls) [reply docs/hubble/tutorials.md]: @sds I'am so sorry! The original Digital Ocean link has been restored as requested. Now is OK! Thanks, please review!

Comment (intls) [reply docs/hubble/tutorials.md]: @sds Could you please help me merge my issue and pull request number #257?

**Pull request merged**